build-scripts/eksd/build.sh

```diff
@@ -11,10 +11,15 @@ else
   (cd "${SNAPCRAFT_PART_BUILD}/eks-distro" && git fetch --all && git pull)
 fi
 
-EKS_TRACK=$(echo "${GIT_TAG}" | cut -f1,2 -d'.' | tr -s . - | cut -c2-)
+EKS_TRACK=$(echo "${KUBE_TRACK}" | cut -f1,2 -d'.' | tr -s . -)
 EKS_RELEASE=$(cat "${SNAPCRAFT_PART_BUILD}"/eks-distro/release/"${EKS_TRACK}"/production/RELEASE)
 
+snapcraftctl set-version "$KUBE_TRACK.$EKS_RELEASE"
+
 curl "https://distro.eks.amazonaws.com/kubernetes-$EKS_TRACK/kubernetes-$EKS_TRACK-eks-$EKS_RELEASE.yaml" -o "${SNAPCRAFT_PART_BUILD}/kubernetes-$EKS_TRACK-eks-$EKS_RELEASE.yaml"
 
 mkdir -p $INSTALL/images
-python3 parse.py "${SNAPCRAFT_PART_BUILD}/kubernetes-$EKS_TRACK-eks-$EKS_RELEASE.yaml" "$INSTALL/images"
+python3 parse-manifest.py "${SNAPCRAFT_PART_BUILD}/kubernetes-$EKS_TRACK-eks-$EKS_RELEASE.yaml" "$INSTALL/images"
+
+export KUBE_VERSION=$(cat "$SNAPCRAFT_PART_BUILD/eksd-components/kubernetes/git-tag")
```

Comment:
This is all we really need, this should be moved to `version.sh`. We can even just fetch it from https://raw.githubusercontent.com/aws/eks-distro/main/projects/kubernetes/kubernetes/1-23/GIT_TAG, no need to clone any repositories.